### PR TITLE
Reduce tests.policy a bit more

### PR DESCRIPTION
**Pull request by rmuir**
Remove some more wildcard permissions and replace with just what we need.

### CODE REVIEW

File: dev-tools/tests.policy [hunk 1]
```diff
@@ -71,8 +71,15 @@ grant {
...
-  permission java.lang.RuntimePermission "accessClassInPackage.*";
-  permission java.lang.RuntimePermission "defineClassInPackage.*";
+
+  // reflection hacks:
+  // needed for Striped64 (what is this doing), also enables unmap hack
+  permission java.lang.RuntimePermission "accessClassInPackage.sun.misc";
+  // needed for mock filesystems in tests (to capture implCloseChannel)
+  permission java.lang.RuntimePermission "accessClassInPackage.sun.nio.ch";
+  // needed by percolation??? (what is going on here)
+  permission java.lang.RuntimePermission "accessClassInPackage.sun.reflect";
```
Comment (s1monw) [inline dev-tools/tests.policy]: WTF what's going on here?

**Comment by s1monw** [comment]
+1 in general - what is Percolation doing here?

**Comment by rmuir** [comment]
I didnt dig into any of these issues. Its important to me to remove the wildcard first, so no more WTFs appear in the meantime unnoticed, and to separate out any long-tail investigations. I planned to open issues for both WTFs.

**Comment by s1monw** [comment]
awesome +1 to push

**Pull request closed without merging**